plugins/backstage-plugin-coder/docs/guides/sdk-recommendations.md

```diff
@@ -0,0 +1,557 @@
+# Using the Coder SDK
+
+This document walks you through using the Coder SDK from within Backstage.
+
+## Disclaimer
+
+As of May 2024, Coder does not have a fully public, versioned SDK published on a package manager like NPM. Coder does intend to release a true JavaScript/TypeScript SDK, but until that is released, the version exposed through Backstage can be best thought of as a "preview"/"testbed" SDK. It is also used in production for [the official Coder VS Code extension](https://github.com/coder/vscode-coder).
```

Comment:
🤔 Hmmm, I wonder if we feel comfortable putting this on our roadmap.

@bpmct What do you think?

Reply:
I'm OK listing it as a roadmap item, let's link to the GitHub issue though: https://github.com/coder/coder/issues/13458

@Parkreiner - feel free to rewrite my issue there too. I did something quickly ^